selectors-4/Overview.bs

```diff
@@ -2295,6 +2295,30 @@ The Focus-Indicated Pseudo-class: '':focus-visible''</h3>
 			apply '':focus-visible''.
 	</div>
 
+	<div class="note">
+		Note: It is a common practice by UAs to always draw a focus indicator on any
```

Comment:
I would replace this with a note about UA stylesheets and `:focus` as the usual mechanism for showing a focus ring, except in a few cases concerning elements which have their default appearance and exhibit something like the behaviour `:focus-visible` should emulate.

Then, follow up with a comment about the eventual goal being to recommend UAs use `:focus-visible` in place of `:focus` so that the default behaviour will be more nuanced.